engine/src/flutter/shell/platform/darwin/ios/framework/Source/FlutterAppDelegate.mm

```diff
@@ -341,4 +341,37 @@ - (int64_t)lastAppModificationTime {
   return [fileDate timeIntervalSince1970];
 }
 
+- (UISceneConfiguration*)application:(UIApplication*)application
+    configurationForConnectingSceneSession:(UISceneSession*)connectingSceneSession
+                                   options:(UISceneConnectionOptions*)options {
+  // Not called before iOS 13.
+  if (@available(iOS 13.0, *)) {
+    NSDictionary* sceneManifest =
+        [[NSBundle mainBundle] objectForInfoDictionaryKey:@"UIApplicationSceneManifest"];
+    NSDictionary* sceneConfigs = sceneManifest[@"UISceneConfigurations"];
+    NSArray* roleConfigs = sceneConfigs[connectingSceneSession.role];
+
+    if (roleConfigs.count > 0) {
+      // Defer to the Info.plist if it is specified there.
+      return nil;
+    }
+
+    // This is the default storyboard in the flutter template.
+    UIStoryboard* storyboard = [UIStoryboard storyboardWithName:@"Main"
+                                                         bundle:[NSBundle mainBundle]];
+    if (!storyboard) {
+      return nil;
+    }
+
+    UISceneConfiguration* config =
+        [UISceneConfiguration configurationWithName:@"flutter"
+                                        sessionRole:connectingSceneSession.role];
+    config.sceneClass = [UIWindowScene class];
+    config.delegateClass = [FlutterAppDelegate class];
+    config.storyboard = storyboard;
+    return config;
+  }
+  return nil;
```

Comment:
oh this `return nil` is for non-iOS 13 case. Can you put available check to the function signature instead of in the body (`API_AVAILABLE`)?

Reply:
Done.  I'm not sure if you have to redeclare the os version since I don't redeclare it in the header but I added it just in case.